data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2090,28 +2160,42 @@
 
         [message]
             speaker=unit
-            message= _ "AH! A ghost!"
+            message= _ "AH! Ghost!"
```

Comment:
I changed it to that because Krash might trigger it and I thought he's more likely to say just "Ghost!" and not "A ghost!" So that seemed more universal to me.